content/ending-frozen-applications.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Ending Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Ending Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to end hanging or frozen processes. The `gnome-system-monitor` provides a GUI process management application with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
+
+>Note: This article details several methods to end processes for a given scenario, but is not exhaustive or guaranteed to provide the most appropriate method for every scenario. Users should also keep in mind that suddenly terminating processes can result in unwanted data loss or system stability issues.
+
+## End Specific Applications or Windows
+
+### Using gnome-system-monitor (GUI)
+
+The gnome-system-monitor comes prepackaged in Pop!_OS. This application provides a GUI interface and functionality similar to the Windows Task Manager.
+
+1. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch a Terminal session.
+2. Enter `gnome-system-monitor` and hit <kbd>Enter</kbd>.
+3. Locate frozen and hanging applications.
+
+   ![gnome-system-monitor](/images/ending-frozen-applications/gnome-system-monitor)
+
+4. Right click the application, click Stop in the dropdown menu.
+
+### Using the kill Command (CLI)
+
+The `kill` command can send a signal termination (SIGTERM) command to any running processes. This process requires knowing the process IDs (PID) of the application(s) to be terminated. The `pidof` command will display the PID(s) associated with any running process.
+
+1. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch a Terminal session.
+2. Type pidof + process or application name and hit Enter.
```

Comment:
Instead of having code blocks beneath each step, you could make any verbatim commands inline code with a single backtick (\`) on each side. If you're using code blocks so it's easier to copy/paste, though, that's perfectly fine.

Reply:
Actually, I didn't read this closely enough. You used the code block for the Firefox example.

Just add a single backtick on either side of `pidof` on this line since that specific word is a verbatim command.

Reply:
done, good catch. I also forgot the <kbd> tags for Enter